common/djangoapps/util/password_policy_validators.py

```diff
@@ -256,7 +256,7 @@ def __init__(self, min_numeric=0):
         self.min_numeric = min_numeric
 
     def validate(self, password, user=None):
-        if _validate_condition(password, lambda c: c.isnumeric(), self.min_numeric):
+        if _validate_condition(password, lambda c: c.isdigit(), self.min_numeric):
```

Comment:
but are both of these actually the same things ?

Reply:
Since we are changing the edx's code, we should make sure that we are doing exactly what we want.

Reply:
Here we are only referring to the `NumericValidator` that checks if the digit is number or not. So, yes `isdigit()` and `isnumeric()` both serve the same purpose here. I have added a Django code Reference link in PR description Django itself uses `isdigit()`.

If you have anything else in mind, let me know I will try that use case.

Reply:
According to this documentation: [Link](https://lerner.co.il/2019/02/17/pythons-str-isdigit-vs-str-isnumeric/#:~:text=isdigit%20only%20returns%20True%20for,it%20contains%20any%20numeric%20characters.&text=It's%20just%20the%20digits%200,used%20in%20place%20of%20digits.)

They can work differently in some scenarios like different languages or Unicode numeric characters.

Reply:
Then i think everything's cool.

Reply:
There are no test failures: 

![image](https://user-images.githubusercontent.com/23108499/84364025-ff2b1f80-abe8-11ea-804a-255a4cb37d5e.png)

Reply:
@danialmalik Created a PR in edX: https://github.com/edx/edx-platform/pull/24197

Let's see. 🤞

Reply:
Booooyaah 😃